cvat/apps/engine/utils.py

```diff
@@ -412,3 +412,7 @@ def directory_tree(path, max_depth=None) -> str:
 
 def is_dataset_export(request: HttpRequest) -> bool:
     return to_bool(request.query_params.get('save_images', False))
+
+def chunked_list(lst, chunk_size):
+    for i in range(0, len(lst), chunk_size):
+        yield lst[i:i + chunk_size]
```

Comment:
Perhaps [take_by](https://github.com/cvat-ai/datumaro/blob/2a4d9dbbd86f2e5fc5f8db2cfd2defdf464e9645/datumaro/util/__init__.py#L60) could be used instead

Reply:
I would prefer to use something internal here
Utility function in 3rdparty library may be renamed/removed and hardly ever it will affect major library version
Internal is just safer

Reply:
I agree, but since our team maintains datumaro that we use, it depends on us. + We already use this method in engine.

Reply:
Until that is not a part of API, it is not safe. 

> We already use this method in engine

That does not mean that such using is correct.

Reply:
Being a part of API means that these methods are at least documented.